sktm/__init__.py

```diff
@@ -284,46 +284,68 @@ def check_patchwork(self):
             new_series = cpw.get_new_patchsets()
             for series in new_series:
                 logging.info("new series: %s", series.get_obj_url_list())
+
+                # Get a list of patch URLs from the series
+                patch_urls = series.get_patch_url_list()
+
+                # Make a list of patch info tuples
+                patches = [self.get_patch_info_from_url(cpw, patch_url)
+                           for patch_url in patch_urls]
+
+                # Add the patches to the database
+                self.db.commit_series(patches)
+
+            # Filter the list of series into the ones we will test (ready)
+            # and the ones that are filtered out (dropped)
             series_ready, series_dropped = self.filter_patchsets(new_series)
+
+            # Log the series that are ready to test
             for series in series_ready:
                 logging.info("ready series: %s", series.get_obj_url_list())
+
+            # Log the series that have been dropped (due to filtering)
             for series in series_dropped:
                 logging.info("dropped series: %s", series.get_obj_url_list())
 
-                # Retrieve all data and save dropped patches in the DB
```

Comment:
Please see issue #100 and commit 0051c4c for explanation why this was added. We don't want to be rechecking patches that were supposed to be dropped all over again until new patch is added to the project. If you want to do it other way feel free to do so but please don't reintroduce the bug :)